cpp/oneapi/dal/algo/basic_statistics/test/batch.cpp

```diff
@@ -41,8 +41,7 @@ TEMPLATE_LIST_TEST_M(basic_statistics_batch_test,
                            te::dataframe_builder{ 500, 250 }.fill_normal(0, 1, 7777),
                            te::dataframe_builder{ 6000, 20 }.fill_normal(-30, 30, 7777),
                            te::dataframe_builder{ 6000, 530 }.fill_normal(-30, 30, 7777),
-                           te::dataframe_builder{ 10000, 200 }.fill_normal(-30, 30, 7777),
-                           te::dataframe_builder{ 1000000, 20 }.fill_normal(-0.5, 0.5, 7777));
+                           te::dataframe_builder{ 10000, 20 }.fill_normal(-0.5, 0.5, 7777));
```

Comment:
Why bigger test cases are removed?

Reply:
I removed the biggest case due to it takes too much time in testing, and we sporadically face with an issue in CI with timeouts of basic statistics. In my understanding there are no code lines which can be really affected by the biggest case, I mean it will reduce the execution time, and will not reduce coverage